inception-ui-kb/src/main/resources/META-INF/asciidoc/user-guide/knowledge_base.adoc

```diff
@@ -0,0 +1,62 @@
+
+[.thumb]
+image::kb4.png[align="center"]
+
+The concept tree on this page is designed using the *subClass* relationship for the configured mapping. Each concept associates itself with a list of instances (in case it has one) on the *Instance* block which appear when we click on a specific concept along with the *Mentions* of the concept in the annotated text. The click on a specific instance shows the panel for the list of statements associated with the instance along with *Mentions* of the instance in the annotated text. In the left bottom side of the page, it list the set of properties from the knowledge base. Clicking on the property showcase the statements associated with the property such as label, domain, range ,etc. 
+
+In case the user has the privilege to edit the knowledge base, the platform allows the user to add statements for concepts, instances and properties. 
+
+==== Knowledge Base Scope setting for feature layer
+
+In the *Projects Settings*, switch to the *Layers* tab where the user can edit the configuration for the pre-defined layers or create a new layer with a click *Create* on the bottom of the panel. The user can define several features for each of the layer as shown in the below figure.
+
+[.thumb]
+image::kb5.png[align="center"]
+
+The feature configuration needs a name along with the *Type* which sets the primary scope of the feature. The dropdown provides the user with one of the option as *KB: Concept/Instance*, which sets the scope of the feature to knowledge base entities. 
+
+Furthermore, the feature configuration, if set to *KB: Concept/Instance* asks for the *Allowed values* which has three values as options: *Any Concept/Instance*, *Only Concept* and *Only Instance*. When the user annotates the text with the particular layer for this feature which has been configures for *Any Concept/Instance* , the auto complete field lists all the concept and instances, while for *Only Concept* and *Only Instance* it list only concept or only instance respectively.
+
+The feature configuration allows the user to set up the configuration to list the values from a specific Knowledge base. 
+
+The *Scope* field sets the configuration of the feature with a specific concept allowing the values for the auto complete field from a specific range of values which includes everything with "is-a" relationship with the specific concept mentioned here. In case the user selects *Any Concept/Instance*, the values will be instances of the specified concept and the child concepts (which includes the hierarchy of child concepts) including their instances, of the specified concept in the *Scope*. Likewise, the choice of *Only Concept* limit the values to the child concepts (which includes the hierarchy of child concepts) and  *Only Instance* limit the values to all of the instances and the instances for the child concepts of the specified concept. 
```

Comment:
I haven't understood the first sentence, it's a little confusing. Try to rephrase it, please. The same for the remaining sentences. 

limit -> limits (twice)